src/internal.c

```diff
@@ -17850,6 +17888,58 @@ static WC_INLINE int EncryptDo(WOLFSSL* ssl, byte* out, const byte* input,
         break;
     #endif /* BUILD_AESGCM || HAVE_AESCCM */
 
+    #ifdef HAVE_ARIA
+        case wolfssl_aria_gcm:
+        {
+            const byte* additionalSrc = input - 5;
```

Comment:
Is it safe to assume input - 5 is valid pointer? Should there be a check? Can we avoid using a hard coded value? Is this the TLS header size `RECORD_HEADER_SZ`?